fuel-toolchain.toml

```diff
@@ -2,5 +2,5 @@
 channel = "latest-2023-10-17"
 
 [components]
-forc = "0.51.1"
+forc = "0.52.1"
 fuel-core = "0.23.0"
```

Comment:
forc v0.52.1 is compatible with fuel-core v0.22.x. forc v0.53.0 was released yesterday and is compatible with fuel-core v0.23.0

Reply:
Updated